### PR TITLE
Add markdown support to OperatorHub and InstalledOperators pages

### PR DESCRIPTION
**Pull request by jhadvig**
Adding support for markdown based on [jira story specification](https://issues.redhat.com/browse/CONSOLE-2294). First I wanted to truncate the blocks using CSS but then Ive realised that its a bit tricky since we are creating multiple paragraph blocks so figured our that best would be to truncated text itself.

/assign @spadgett

### CODE REVIEW

**Comment by jhadvig** [comment]
Screens:
- Screenshot 2020-06-25 at 20 11 37

- Screenshot 2020-06-25 at 20 11 03

- Screenshot 2020-06-25 at 20 11 13

File: frontend/public/components/markdown-view.tsx [hunk 1]
```diff
@@ -89,13 +89,19 @@ export class SyncMarkdownView extends React.Component<
         <link rel="stylesheet" href="${link.href}">`,
       '',
     );
+    const content = this.props.truncateContent
+      ? _.truncate(this.props.content, {
```
Comment (spadgett) [inline frontend/public/components/markdown-view.tsx:93]: It would be better to use CSS line clamping. See the `co-line-clamp` mixin.

Comment (jhadvig) [reply frontend/public/components/markdown-view.tsx:93]: @spadgett  I've already tried this approach, by adding `co-line-clamp` to the main div.
Screenshot 2020-06-26 at 12 52 21

The issue is with the multiple paragraph blocks

Screenshot 2020-06-26 at 12 47 47

For that reason I went with the programatic solution here. Other solution would be to tweak the CSS for the markdown iframe, but would rather confirm that approach before starting with the work.

File: frontend/public/components/markdown-view.tsx [hunk 2]
```diff
@@ -89,13 +89,19 @@ export class SyncMarkdownView extends React.Component<
...
+          length: 100,
```
Comment (spadgett) [inline frontend/public/components/markdown-view.tsx:94]: 100 feels a little too short. Maybe 256?

File: frontend/public/components/markdown-view.tsx [hunk 3]
```diff
@@ -89,13 +89,19 @@ export class SyncMarkdownView extends React.Component<
...
+          separator: ' ',
```
Comment (spadgett) [inline frontend/public/components/markdown-view.tsx:95]:
```suggestion
          separator: ' ',
          omission: '\u2026', // ellipsis character
```

File: frontend/public/components/markdown-view.tsx (hunk 2 above)
Comment (jhadvig) [reply frontend/public/components/markdown-view.tsx:94]: I wanted to leave it to approx. 3 lines, which is about 100+- chars

File: frontend/public/components/markdown-view.tsx (hunk 3 above)
Comment (jhadvig) [reply frontend/public/components/markdown-view.tsx:95]: [_.truncate](https://lodash.com/docs/4.17.15#truncate) already defaults to the ellipsis character

Comment (spadgett) [reply frontend/public/components/markdown-view.tsx:95]: > [_.truncate](https://lodash.com/docs/4.17.15#truncate) already defaults to the ellipsis character

It defaults to `...` which is not the same as the Unicode ellipsis.

File: frontend/public/components/markdown-view.tsx (hunk 2 above)
Comment (spadgett) [reply frontend/public/components/markdown-view.tsx:94]: > I wanted to leave it to approx. 3 lines, which is about 100+- chars

I don't think it's enough content to describe some resources. We're really trying to catch the extreme cases where there are paragraphs of text.

**Comment by jhadvig** [comment]
@spadgett comments addressed. PTAL

**Comment by jhadvig** [comment]
@spadgett I think we are good to go here :)

**Pull request merged**